### DISCUSSION TITLE
Edit articles on the fronted

### DISCUSSION THREAD

**Discussion by Dnsftl** [templaza/astroid-framework#545] [General]
Hi,

Did a clean install on a dev.domain with joomla 5.0.2, Jollyany - standard 3.5.21 and Astroid 3.0.6  updated to 3.0.9, as Editor JCE pro version 2.9.61

Now it appears that after logging in you can no longer edit articles on the fronted, switch to Cassiopeia, astroid one or
astroid zero, article editing is possible. Is it due to a bug in JollyAny?

Thanks in advance,

Dennis

**Comment by Chacapamac** [comment]
Just to follow conversation.

**Comment by sonvnn** [comment]
@Dnsftl Yes, It's issue of Jollyany Template. I will soon update Jollyany Classic please wait for the update. 

Thanks,

**Comment by Dnsftl** [comment]
Okay, we are waiting for the update. 

Thanks
